lib/cf/bridge/src/index.js

```diff
@@ -293,7 +293,9 @@ const stopReporting = () => {
 };
 
 const scheduleUsageReporting = () => {
-  oauth.start('bridge', 'secret');
+  var clientId = process.env.CLIENT_ID ? process.env.CLIENT_ID : "bridge";
```

Comment:
This was the easiest way to have the bridge running. We had the idea to make this configurable and use the oauth module in abacus (https://www.pivotaltracker.com/story/show/108061970).